tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -931,10 +932,35 @@ Ghost
         char_fx.color.a = alpha
         return true
 
+ .. code-tab:: csharp
+ 
+    using Godot;
+
+    [Tool]
+    public class RichTextGhost : RichTextEffect
+    {
+        // Syntax: [ghost freq=5.0 span=10.0][/ghost]
+
+        // Define the tag name.
+        private const string Bbcode = "ghost";
+
+        public override bool _ProcessCustomFx(CharFXTransform charFX)
+        {
+            // Get parameters, or use the provided default value if missing.
+            var speed = charFx.Env.Get("freq", 5.0f);
+            var span = charFx.Env.Get("span", 10.0f);
```

Comment:
`Godot.Dictionary` doesn't implement a `Get` method. Use the extension method `GetValueOrDefault`. And cast to `float` because the dictionary returned by `Env` is untyped.

```suggestion
            var speed = (float)charFx.Env.GetValueOrDefault("freq", 5.0f);
            var span = (float)charFx.Env.GetValueOrDefault("span", 10.0f);
```